src/CaptureClient/include/CaptureClient/CaptureClient.h

```diff
@@ -32,6 +32,34 @@
 
 namespace orbit_capture_client {
 
+struct CaptureClientOptions {
```

Comment:
The word order is strange, I think this should be called `ClientCaptureOptions` (the name @beckerhe used without noticing it's different, and also as counterpart to `ToGrpcCaptureOptions`).